exercises/rect/rect_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestRect(t *testing.T) {
+	for _, tc := range []struct {
+		x, y int
+		want [][]int
+	}{
+		{0, 0, [][]int{}},
+		{7, 8, [][]int{
+			{1, 1, 1, 1, 1, 1, 1},
+			{1, 0, 0, 0, 0, 0, 1},
+			{1, 0, 0, 0, 0, 0, 1},
+			{1, 0, 0, 0, 0, 0, 1},
+			{1, 0, 0, 0, 0, 0, 1},
+			{1, 0, 0, 0, 0, 0, 1},
+			{1, 0, 0, 0, 0, 0, 1},
+			{1, 1, 1, 1, 1, 1, 1},
+		}},
+		{5, 5, [][]int{
+			{1, 1, 1, 1, 1},
+			{1, 0, 0, 0, 1},
+			{1, 0, 0, 0, 1},
+			{1, 0, 0, 0, 1},
+			{1, 1, 1, 1, 1},
+		}},
+		{9, 3, [][]int{
```

Comment:
The dimensions are swapped. There should be 9 rows and 3 columns instead (you have 9 columns and 3 rows).

Reply:
Actually, no. Please disregard the comment above.